smartautoclicker/src/main/res/layout/dialog_permissions.xml

```diff
@@ -15,6 +15,11 @@
  * You should have received a copy of the GNU General Public License
  * along with this program; If not, see <http://www.gnu.org/licenses/>.
 -->
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
xmlns elements should always be declared on the first element, so:
* xmlns:android should be removed from here as you declared it above
* xmlns:tools should be moved in the ScrollView element

Reply:
OK, thank you so much for your advice, and I have changed this part according to your comments.